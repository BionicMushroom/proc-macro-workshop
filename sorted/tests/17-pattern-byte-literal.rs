#[sorted::check]
fn main() {
    let x = b'a';

    #[sorted]
    match x {
        b'd' => println!("d"),
        b'a' => println!("a"),
        _ => println!("other"),
    }
}
