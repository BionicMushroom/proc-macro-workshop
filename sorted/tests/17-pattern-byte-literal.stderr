error: b'a' should sort before b'd'
 --> tests/17-pattern-byte-literal.rs:8:9
  |
8 |         b'a' => println!("a"),
  |         ^^^^
